codegen/codegen.sh

```diff
@@ -65,7 +65,8 @@ for type in "${rekor_types[@]}"; do
     datamodel-codegen \
         --input "${rekor_dir}/pkg/types/${type}/${type}_schema.json" \
         --input-file-type jsonschema \
-        --target-python-version 3.8 \
+        --target-python-version 3.9 \
+        --formatters black isort ruff-check ruff-format \
```

Comment:
We can now ask datamodel-codegen to use the ruff formatters.

Reply:
Removing `isort` has no effect, but ruff seems to not mind longer lines. And since the default option included `black`, maybe it's better to keep it, so we don't have a big diff in the autogenerated PR.

```diff
-    content: str = Field(
-        ...,
-        description="Specifies the package inline within the document",
-    )
+    content: str = Field(..., description="Specifies the package inline within the document")
```

Reply:
I think because of https://github.com/ramonpetgrave64/sigstore-rekor-types/blob/51e11621492c4c62fc7f656107734eba8a756538/pyproject.toml#L63-L64

Reply:
nevermind, it's fine without `black`